app/classes/Transvision/ShowResults.php

```diff
@@ -386,10 +423,12 @@ public static function resultsTable($search_object, $search_results, $page)
             } elseif (! $target_string) {
                 $target_string = '<em class="error">Warning: Empty string</em>';
             } else {
-                $meta_target = "
-                      <span class='clipboard' data-clipboard-target='#{$clipboard_target_string}' alt='Copy to clipboard'><img src='/img/copy_icon_black_18x18.png'></span>
-                      {$error_message}";
+                $meta_target = "<span class='clipboard' data-clipboard-target='#{$regular_string_id}' alt='Copy to clipboard'><img src='/img/copy_icon_black_18x18.png'></span>";
+                if ($transliterate) {
+                    $meta_target .= "<input type='button' value='To latin' data-transliterated-id='{$string_id}' class='transliterate_button'>";
+                }
             }
+            $meta_target .= "{$error_message}";
```

Comment:
This line should be inside the else statement above, since we want to display the error only when the string is not empty. Since it’s only concatenating a variable, it should be `$meta_target .= $error_message;`